spec/unit/sandbox/path_list_spec.rb

```diff
@@ -336,12 +336,20 @@ module Pod
         @path_list.dirs.map(&:to_s).should.include?('Classes/symlinked')
       end
 
-      it 'doesn\'t include file in symlinked dir' do
+      it 'includes file in symlinked dir if link to other directory which not in root' do
         @path_list.instance_variable_set(:@files, nil)
         @path_list.instance_variable_set(:@dirs, nil)
         File.symlink(@tmpdir, @symlink_dir)
 
-        @path_list.files.map(&:to_s).should.not.include?('Classes/symlinked/someheader.h')
+        @path_list.files.map(&:to_s).should.include?('Classes/symlinked/someheader.h')
```

Comment:
Will this affect headers in vendored frameworks?

Reply:
It is work for local development pods: `pod 'xx', :path=>'../xxx'`